pkg/apis/pgv2.percona.com/v2/perconapgcluster_types.go

```diff
@@ -247,6 +253,11 @@ func (cr *PerconaPGCluster) ToCrunchy(ctx context.Context, postgresCluster *crun
 			annotations[pNaming.ToCrunchyAnnotation(k)] = v
 		}
 	}
+
+	if cr.Spec.AutoCreateUserSchema != nil && *cr.Spec.AutoCreateUserSchema {
+		annotations[naming.AutoCreateUserSchemaAnnotation] = "true"
+	}
+
```

Comment:
we don't need here check for operator version?

Reply:
No, don't think so.